### ISSUE TITLE
Error after installation - drush: command not found

### ISSUE THREAD

**Issue by elsatch** [kalamuna/kalaboxv1#122]
### Kalabox Log

2014-08-24T22:47:04.233Z - info: Box initialized.
2014-08-24T22:47:04.237Z - info: Kalabox Version: 1.0-beta7
2014-08-24T22:47:04.237Z - info: Mac OS Version: 10.9.4

2014-08-24T22:47:04.238Z - info: VirtualBox Version: false
2014-08-24T22:47:04.238Z - info: Vagrant Version: false
2014-08-24T22:48:03.034Z - info: VirtualBox not installed or wrong version. Installing.
2014-08-24T22:48:03.037Z - info: User granted permission to install VirtualBox
2014-08-24T22:49:03.409Z - info: Successfully downloaded VirtualBox DMG.
2014-08-24T22:49:29.569Z - info: Successfully mounted VirtualBox DMG.
2014-08-24T22:50:06.992Z - info: VirtualBox installation complete.
2014-08-24T22:50:08.475Z - info: Successfully ejected VirtualBox DMG.
2014-08-24T22:50:08.489Z - info: Vagrant not installed or wrong version. Installing.
2014-08-24T22:50:08.491Z - info: User granted permission to install Vagrant
2014-08-24T22:50:24.254Z - info: Successfully downloaded Vagrant DMG.
2014-08-24T22:50:32.266Z - info: Successfully mounted Vagrant DMG.
2014-08-24T22:50:45.532Z - info: Vagrant installation complete.
2014-08-24T22:50:45.868Z - info: Successfully ejected Vagrant DMG.
2014-08-24T22:52:09.180Z - info: Box image downloaded.
2014-08-24T22:52:09.843Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T22:52:12.826Z - info: Kalastack downloaded
2014-08-24T22:52:12.828Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T22:52:36.505Z - info: Adding base box image to Vagrant.
2014-08-24T22:52:50.409Z - info: Wrote configuration file.
2014-08-24T22:52:50.610Z - info: Executing box spinup.
2014-08-24T22:52:50.725Z - info: Starting spinup attempt 0
2014-08-24T23:42:21.252Z - info: Box initialized.
2014-08-24T23:42:21.259Z - info: Kalabox Version: 1.0-beta7
2014-08-24T23:42:21.311Z - info: Mac OS Version: 10.9.4

2014-08-24T23:42:21.311Z - info: VirtualBox Version: 4.3.6r91406

2014-08-24T23:42:21.351Z - info: Vagrant Version: Vagrant 1.4.3

2014-08-24T23:42:21.360Z - info: Installation completed successfully.
2014-08-24T23:42:34.968Z - info: Existing VirtualBox installation, version 4.3.6
2014-08-24T23:42:34.971Z - info: Existing Vagrant installation, version 1.4.3
2014-08-24T23:42:34.991Z - info: Kalabox image already downloaded.
2014-08-24T23:42:34.992Z - info: Box image downloaded.
2014-08-24T23:42:35.126Z - info: Kalastack already downloaded
2014-08-24T23:42:35.127Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T23:42:37.814Z - warn: Error retrieving sites list: Error: getaddrinfo ENOTFOUND
2014-08-24T23:42:53.943Z - info: Kalastack downloaded
2014-08-24T23:42:53.945Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T23:43:20.086Z - info: Wrote configuration file.
2014-08-24T23:43:20.204Z - info: Executing box spinup.
2014-08-24T23:43:20.303Z - info: Starting spinup attempt 0
2014-08-24T23:43:38.992Z - info: Box initialized.
2014-08-24T23:43:38.993Z - info: Kalabox Version: 1.0-beta7
2014-08-24T23:43:38.993Z - info: Mac OS Version: 10.9.4

2014-08-24T23:43:38.993Z - info: VirtualBox Version: 4.3.6r91406

2014-08-24T23:43:38.993Z - info: Vagrant Version: Vagrant 1.4.3

2014-08-24T23:43:38.995Z - info: Installation completed successfully.
2014-08-24T23:45:41.569Z - warn: Error retrieving sites list: Error: getaddrinfo ENOTFOUND
2014-08-24T23:47:13.879Z - warn: Error retrieving sites list: Error: connect ECONNREFUSED
2014-08-24T23:47:19.353Z - error: An error occurred in an async call.
cause stack is ...
  Error: Command failed: bash: drush: command not found

```
at ChildProcess.exithandler (child_process.js:661:12)
at ChildProcess.EventEmitter.emit (events.js:101:17)
at maybeClose (child_process.js:773:16)
at Socket.eval (child_process.js:994:11)
at Socket.EventEmitter.emit (events.js:98:17)
at Pipe.eval [as close] (net.js:459:12)
```

async call history is ...
 [ { function: 'startBox4',
    location: '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.lY24yP/kalabox/box.js:168:108',
    mapping: { description: 'tail' } } ]

Note that I have the following setup in my computer to support a local MAMP installation: 
Computer:~ satch$ ls -la /usr/local/bin/drush
lrwxr-xr-x  1 root  wheel  47  7 ago 19:34 /usr/local/bin/drush -> /Users/satch/.composer/vendor/drush/drush/drush 
##

**Comment by pirog** [comment]
so kalabox v1 will only install `drush` inside the virtual machine. click on the SSH button on the dash and then try running it in the terminal that pops up.

**Comment by elsatch** [comment]
Hi @pirog,

That was exactly my same thought. I've just logged in into the vm using ssh. This is the output:

The authenticity of host '[127.0.0.1]:2222 ([127.0.0.1]:2222)' can't be established.
RSA key fingerprint is 50:db:75:ba:11:2f:43:c9:ab:14:40:6d:7f:a1:ee:e3.
Are you sure you want to continue connecting (yes/no)? yes
Warning: Permanently added '[127.0.0.1]:2222' (RSA) to the list of known hosts.
Welcome to Ubuntu 12.04 LTS (GNU/Linux 3.2.0-23-generic x86_64)
- Documentation:  https://help.ubuntu.com/
  New release '14.04.1 LTS' available.
  Run 'do-release-upgrade' to upgrade to it.

Welcome to your Vagrant-built virtual machine.
Last login: Fri Sep 14 06:23:18 2012 from 10.0.2.2
vagrant@kala:~$ drush
The program 'drush' is currently not installed.  You can install it by typing:
sudo apt-get install drush
vagrant@kala:~$ sudo apt-get install drush
Reading package lists... Done
Building dependency tree  
Reading state information... Done
The following extra packages will be installed:
  autoconf automake autotools-dev libltdl-dev libltdl7 libmysqlclient18
  libtool m4 mysql-common php-console-table php-pear php5-cli php5-common
  php5-dev php5-mysql shtool
Suggested packages:
  autoconf2.13 autoconf-archive gnu-standards autoconf-doc gettext
  mysql-client libtool-doc automaken gfortran fortran95-compiler gcj
  php5-suhosin
The following NEW packages will be installed:
  autoconf automake autotools-dev drush libltdl-dev libltdl7 libmysqlclient18
  libtool m4 mysql-common php-console-table php-pear php5-cli php5-common
  php5-dev php5-mysql shtool
0 upgraded, 17 newly installed, 0 to remove and 169 not upgraded.
Need to get 8,019 kB/8,981 kB of archives.
After this operation, 28.0 MB of additional disk space will be used.
Do you want to continue [Y/n]? 
Get:1 http://us.archive.ubuntu.com/ubuntu/ precise/main libltdl7 amd64 2.4.2-1ubuntu1 [37.6 kB]
Get:2 http://us.archive.ubuntu.com/ubuntu/ precise/main m4 amd64 1.4.16-2ubuntu1 [200 kB]
Get:3 http://us.archive.ubuntu.com/ubuntu/ precise/main autoconf all 2.68-1ubuntu2 [560 kB]
Get:4 http://us.archive.ubuntu.com/ubuntu/ precise/main autotools-dev all 20120210.1ubuntu1 [42.4 kB]
Get:5 http://us.archive.ubuntu.com/ubuntu/ precise/main automake all 1:1.11.3-1ubuntu2 [571 kB]
Get:6 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php5-common amd64 5.3.10-1ubuntu3.13 [1,774 kB]
Get:7 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php5-cli amd64 5.3.10-1ubuntu3.13 [3,051 kB]
Get:8 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php-pear all 5.3.10-1ubuntu3.13 [371 kB]
Get:9 http://us.archive.ubuntu.com/ubuntu/ precise/universe php-console-table all 1.1.4-1 [14.5 kB]
Get:10 http://us.archive.ubuntu.com/ubuntu/ precise/universe drush all 4.5-6 [242 kB]
Get:11 http://us.archive.ubuntu.com/ubuntu/ precise/main libltdl-dev amd64 2.4.2-1ubuntu1 [202 kB]
Get:12 http://us.archive.ubuntu.com/ubuntu/ precise/main libtool amd64 2.4.2-1ubuntu1 [302 kB]
Get:13 http://us.archive.ubuntu.com/ubuntu/ precise/main shtool all 2.0.8-6 [149 kB]
Get:14 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php5-dev amd64 5.3.10-1ubuntu3.13 [424 kB]
Get:15 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php5-mysql amd64 5.3.10-1ubuntu3.13 [76.6 kB]
Fetched 8,019 kB in 8s (983 kB/s)  
Selecting previously unselected package libltdl7.
(Reading database ... 51754 files and directories currently installed.)
Unpacking libltdl7 (from .../libltdl7_2.4.2-1ubuntu1_amd64.deb) ...
Selecting previously unselected package mysql-common.
Unpacking mysql-common (from .../mysql-common_5.5.38-0ubuntu0.12.04.1_all.deb) ...
Selecting previously unselected package libmysqlclient18.
Unpacking libmysqlclient18 (from .../libmysqlclient18_5.5.38-0ubuntu0.12.04.1_amd64.deb) ...
Selecting previously unselected package m4.
Unpacking m4 (from .../m4_1.4.16-2ubuntu1_amd64.deb) ...
Selecting previously unselected package autoconf.
Unpacking autoconf (from .../autoconf_2.68-1ubuntu2_all.deb) ...
Selecting previously unselected package autotools-dev.
Unpacking autotools-dev (from .../autotools-dev_20120210.1ubuntu1_all.deb) ...
Selecting previously unselected package automake.
Unpacking automake (from .../automake_1%3a1.11.3-1ubuntu2_all.deb) ...
Selecting previously unselected package php5-common.
Unpacking php5-common (from .../php5-common_5.3.10-1ubuntu3.13_amd64.deb) ...
Selecting previously unselected package php5-cli.
Unpacking php5-cli (from .../php5-cli_5.3.10-1ubuntu3.13_amd64.deb) ...
Selecting previously unselected package php-pear.
Unpacking php-pear (from .../php-pear_5.3.10-1ubuntu3.13_all.deb) ...
Selecting previously unselected package php-console-table.
Unpacking php-console-table (from .../php-console-table_1.1.4-1_all.deb) ...
Selecting previously unselected package drush.
Unpacking drush (from .../archives/drush_4.5-6_all.deb) ...
Selecting previously unselected package libltdl-dev.
Unpacking libltdl-dev (from .../libltdl-dev_2.4.2-1ubuntu1_amd64.deb) ...
Selecting previously unselected package libtool.
Unpacking libtool (from .../libtool_2.4.2-1ubuntu1_amd64.deb) ...
Selecting previously unselected package shtool.
Unpacking shtool (from .../shtool_2.0.8-6_all.deb) ...
Selecting previously unselected package php5-dev.
Unpacking php5-dev (from .../php5-dev_5.3.10-1ubuntu3.13_amd64.deb) ...
Selecting previously unselected package php5-mysql.
Unpacking php5-mysql (from .../php5-mysql_5.3.10-1ubuntu3.13_amd64.deb) ...
Processing triggers for install-info ...
Processing triggers for man-db ...
Setting up libltdl7 (2.4.2-1ubuntu1) ...
Setting up mysql-common (5.5.38-0ubuntu0.12.04.1) ...
Setting up libmysqlclient18 (5.5.38-0ubuntu0.12.04.1) ...
Setting up m4 (1.4.16-2ubuntu1) ...
Setting up autoconf (2.68-1ubuntu2) ...
Setting up autotools-dev (20120210.1ubuntu1) ...
Setting up automake (1:1.11.3-1ubuntu2) ...
update-alternatives: using /usr/bin/automake-1.11 to provide /usr/bin/automake (automake) in auto mode.
Setting up php5-common (5.3.10-1ubuntu3.13) ...
Setting up php5-cli (5.3.10-1ubuntu3.13) ...

Creating config file /etc/php5/cli/php.ini with new version
update-alternatives: using /usr/bin/php5 to provide /usr/bin/php (php) in auto mode.
Setting up php-pear (5.3.10-1ubuntu3.13) ...
Setting up php-console-table (1.1.4-1) ...
Setting up drush (4.5-6) ...
Setting up libltdl-dev (2.4.2-1ubuntu1) ...
Setting up libtool (2.4.2-1ubuntu1) ...
Setting up shtool (2.0.8-6) ...
Setting up php5-dev (5.3.10-1ubuntu3.13) ...
update-alternatives: using /usr/bin/php-config5 to provide /usr/bin/php-config (php-config) in auto mode.
update-alternatives: using /usr/bin/phpize5 to provide /usr/bin/phpize (phpize) in auto mode.
Setting up php5-mysql (5.3.10-1ubuntu3.13) ...
Processing triggers for libc-bin ...
ldconfig deferred processing now taking place
vagrant@kala:~$ drush
Execute a drush command. Run `drush help [command]` to view command-specific
help.  Run `drush topic` to read even more documentation.

Global options (see `drush topic core-global-options` for the full list).:
 -r <path>, --root=<path>                  Drupal root directory to use  
                                           (default: current directory)  
 -l http://example.com:8888,               URI of the drupal site to use (only  
 --uri=http://example.com:8888             needed in multisite environments or  
                                           when running on an alternate port)  
 -v, --verbose                             Display extra information about the  
                                           command.  
 -d, --debug                               Display even more information,  
                                           including internal messages.  
 -y, --yes                                 Assume 'yes' as answer to all  
                                           prompts  
 -n, --no                                  Assume 'no' as answer to all prompts 
 -s, --simulate                            Simulate all relevant actions (don't 
                                           actually change the system)  
 -p, --pipe                                Emit a compact representation of the 
                                           command for scripting.  
 -h, --help                                This help system.  
 --version                                 Show drush version.  
 --php                                     The absolute path to your PHP  
                                           intepreter, if not 'php' in the  
                                           path.                                

Core drush commands: (core)
 archive-dump (ard,    Backup your code, files, and database into a single  
 archive-backup, arb)  file.  
 archive-restore       Expand a site archive into a Drupal web site.  
 (arr)  
 cache-clear (cc)      Clear a specific cache, or all drupal caches.  
 cache-get (cg)        Fetch a cached object and display it.  
 cache-set (cs)        Cache an object expressed in JSON or var_export()  
                       format.  
 core-cli (cli)        Enter a new shell optimized for drush use.  
 core-cron (cron)      Run all cron hooks in all active modules for specified  
                       site.  
 core-rsync (rsync)    Rsync the Drupal tree to/from another server using ssh.  
 core-status (status,  Provides a birds-eye view of the current Drupal  
 st)                   installation, if any.  
 core-topic (topic)    Read detailed documentation on a given topic.  
 drupal-directory      Return path to a given module/theme directory.  
 (dd)  
 help                  Print this help message. See `drush help help` for more  
                       options.  
 image-flush           Flush all derived images for a given style.  
 php-eval (eval, ev)   Evaluate arbitrary php code after bootstrapping Drupal  
                       (if available).  
 php-script (scr)      Run php script(s).  
 search-index          Index the remaining search items without wiping the  
                       index.  
 search-reindex        Force the search index to be rebuilt.  
 search-status         Show how many items remain to be indexed out of the  
                       total.  
 self-update           Update drush to the latest version, if available.  
 (selfupdate)  
 site-alias (sa)       Print site alias records for all known site aliases and  
                       local sites.  
 site-install (si)     Install Drupal along with modules/themes/configuration  
                       using the specified install profile.  
 site-upgrade (sup)    Run a major version upgrade for Drupal (e.g. Drupal 6 to 
                       Drupal 7).  A copy of the site is made, and then  
                       upgraded; the source site is not changed.  
 test-clean            Clean temporary tables and files.  
 test-run              Run tests. Note that you must use the --uri option.  
 updatedb (updb)       Apply any database updates required (as with running  
                       update.php).  
 variable-delete       Delete a variable.  
 (vdel)  
 variable-get (vget)   Get a list of some or all site variables and values.  
 variable-set (vset)   Set a variable.  
 version               Show drush version.  
 watchdog-delete       Delete watchdog messages.  
 (wd-del, wd-delete)  
 watchdog-list         Show available message types and severity levels. A  
 (wd-list)             prompt will ask for a choice to show watchdog messages.  
 watchdog-show         Show watchdog messages.  
 (wd-show, ws)                                                                  

Field commands: (field)
 field-clone           Clone a field and all its instances.  
 field-create          Create fields and instances. Returns urls for field  
                       editing.  
 field-delete          Delete a field and its instances.  
 field-info            View information about fields, field_types, and widgets. 
 field-update          Return URL for field editing web page.                   

Project manager commands: (pm)
 pm-disable (dis)      Disable one or more extensions (modules or themes).  
 pm-download (dl)      Download projects from drupal.org or other sources.  
 pm-enable (en)        Enable one or more extensions (modules or themes).  
 pm-info (pmi)         Show detailed info for one or more extensions (modules 
                       or themes).  
 pm-list (pml)         Show a list of available extensions (modules and  
                       themes).  
 pm-refresh (rf)       Refresh update status information.  
 pm-releasenotes       Print release notes for given projects.  
 (rln)  
 pm-releases (rl)      Print release information for given projects.  
 pm-uninstall          Uninstall one or more modules.  
 pm-update (up)        Update Drupal core and contrib projects and apply any  
                       pending database updates (Same as pm-updatecode +  
                       updatedb).  
 pm-updatecode (upc)   Update Drupal core and contrib projects to latest  
                       recommended releases.                                  

SQL commands: (sql)
 sql-cli (sqlc)        Open a SQL command-line interface using Drupal's  
                       credentials.  
 sql-connect           A string for connecting to the DB.  
 sql-drop              Drop all tables in a given database.  
 sql-dump              Exports the Drupal DB as SQL using mysqldump or  
                       equivalent.  
 sql-query (sqlq)      Execute a query against the site database.  
 sql-sync              Copy and import source database to target database. 
                       Transfers via rsync.                                

User commands: (user)
 user-add-role (urol)  Add a role to the specified user accounts.  
 user-block (ublk)     Block the specified user(s).  
 user-cancel (ucan)    Cancel a user account with the specified name.  
 user-create (ucrt)    Create a user account with the specified name.  
 user-information      Print information about the specified user(s).  
 (uinf)  
 user-login (uli)      Display a one time login link for the given user account 
                       (defaults to uid 1).  
 user-password (upwd)  (Re)Set the password for the user account with the  
                       specified name.  
 user-remove-role      Remove a role from the specified user accounts.  
 (urrol)  
 user-unblock (uublk)  Unblock the specified user(s).                           

vagrant@kala:~$ 

This is the error log once I stop and start kalabox again.

### Kalabox Log

2014-08-24T22:47:04.233Z - info: Box initialized.
2014-08-24T22:47:04.237Z - info: Kalabox Version: 1.0-beta7
2014-08-24T22:47:04.237Z - info: Mac OS Version: 10.9.4

2014-08-24T22:47:04.238Z - info: VirtualBox Version: false
2014-08-24T22:47:04.238Z - info: Vagrant Version: false
2014-08-24T22:48:03.034Z - info: VirtualBox not installed or wrong version. Installing.
2014-08-24T22:48:03.037Z - info: User granted permission to install VirtualBox
2014-08-24T22:49:03.409Z - info: Successfully downloaded VirtualBox DMG.
2014-08-24T22:49:29.569Z - info: Successfully mounted VirtualBox DMG.
2014-08-24T22:50:06.992Z - info: VirtualBox installation complete.
2014-08-24T22:50:08.475Z - info: Successfully ejected VirtualBox DMG.
2014-08-24T22:50:08.489Z - info: Vagrant not installed or wrong version. Installing.
2014-08-24T22:50:08.491Z - info: User granted permission to install Vagrant
2014-08-24T22:50:24.254Z - info: Successfully downloaded Vagrant DMG.
2014-08-24T22:50:32.266Z - info: Successfully mounted Vagrant DMG.
2014-08-24T22:50:45.532Z - info: Vagrant installation complete.
2014-08-24T22:50:45.868Z - info: Successfully ejected Vagrant DMG.
2014-08-24T22:52:09.180Z - info: Box image downloaded.
2014-08-24T22:52:09.843Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T22:52:12.826Z - info: Kalastack downloaded
2014-08-24T22:52:12.828Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T22:52:36.505Z - info: Adding base box image to Vagrant.
2014-08-24T22:52:50.409Z - info: Wrote configuration file.
2014-08-24T22:52:50.610Z - info: Executing box spinup.
2014-08-24T22:52:50.725Z - info: Starting spinup attempt 0
2014-08-24T23:42:21.252Z - info: Box initialized.
2014-08-24T23:42:21.259Z - info: Kalabox Version: 1.0-beta7
2014-08-24T23:42:21.311Z - info: Mac OS Version: 10.9.4

2014-08-24T23:42:21.311Z - info: VirtualBox Version: 4.3.6r91406

2014-08-24T23:42:21.351Z - info: Vagrant Version: Vagrant 1.4.3

2014-08-24T23:42:21.360Z - info: Installation completed successfully.
2014-08-24T23:42:34.968Z - info: Existing VirtualBox installation, version 4.3.6
2014-08-24T23:42:34.971Z - info: Existing Vagrant installation, version 1.4.3
2014-08-24T23:42:34.991Z - info: Kalabox image already downloaded.
2014-08-24T23:42:34.992Z - info: Box image downloaded.
2014-08-24T23:42:35.126Z - info: Kalastack already downloaded
2014-08-24T23:42:35.127Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T23:42:37.814Z - warn: Error retrieving sites list: Error: getaddrinfo ENOTFOUND
2014-08-24T23:42:53.943Z - info: Kalastack downloaded
2014-08-24T23:42:53.945Z - info: Installing Vagrant plugin vagrant-hostsupdater
2014-08-24T23:43:20.086Z - info: Wrote configuration file.
2014-08-24T23:43:20.204Z - info: Executing box spinup.
2014-08-24T23:43:20.303Z - info: Starting spinup attempt 0
2014-08-24T23:43:38.992Z - info: Box initialized.
2014-08-24T23:43:38.993Z - info: Kalabox Version: 1.0-beta7
2014-08-24T23:43:38.993Z - info: Mac OS Version: 10.9.4

2014-08-24T23:43:38.993Z - info: VirtualBox Version: 4.3.6r91406

2014-08-24T23:43:38.993Z - info: Vagrant Version: Vagrant 1.4.3

2014-08-24T23:43:38.995Z - info: Installation completed successfully.
2014-08-24T23:45:41.569Z - warn: Error retrieving sites list: Error: getaddrinfo ENOTFOUND
2014-08-24T23:47:13.879Z - warn: Error retrieving sites list: Error: connect ECONNREFUSED
2014-08-24T23:47:19.353Z - error: An error occurred in an async call.
cause stack is ...
  Error: Command failed: bash: drush: command not found

```
at ChildProcess.exithandler (child_process.js:661:12)
at ChildProcess.EventEmitter.emit (events.js:101:17)
at maybeClose (child_process.js:773:16)
at Socket.eval (child_process.js:994:11)
at Socket.EventEmitter.emit (events.js:98:17)
at Pipe.eval [as close] (net.js:459:12)
```

async call history is ...
 [ { function: 'startBox4',
    location: '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.lY24yP/kalabox/box.js:168:108',
    mapping: { description: 'tail' } } ]
2014-08-25T22:44:04.056Z - info: Box initialized.
2014-08-25T22:44:04.061Z - info: Kalabox Version: 1.0-beta7
2014-08-25T22:44:04.062Z - info: Mac OS Version: 10.9.4

2014-08-25T22:44:04.062Z - info: VirtualBox Version: 4.3.6r91406

2014-08-25T22:44:04.062Z - info: Vagrant Version: Vagrant 1.4.3

2014-08-25T22:44:07.558Z - warn: Error retrieving sites list: Error: connect ECONNREFUSED
2014-08-25T22:47:20.693Z - warn: Box has no Internet connection.
2014-08-25T22:53:27.459Z - warn: Error retrieving sites list: Error: connect ECONNREFUSED
2014-08-25T22:53:31.621Z - error: An error occurred in an async call.
cause stack is ...
  Error: Command failed: bash: drush: command not found

```
at ChildProcess.exithandler (child_process.js:661:12)
at ChildProcess.EventEmitter.emit (events.js:101:17)
at maybeClose (child_process.js:773:16)
at Socket.eval (child_process.js:994:11)
at Socket.EventEmitter.emit (events.js:98:17)
at Pipe.eval [as close] (net.js:459:12)
```

async call history is ...
 [ { function: 'startBox4',
    location: '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.sK7HIv/kalabox/box.js:168:108',
    mapping: { description: 'tail' } } ]
2014-08-26T15:14:47.087Z - info: Box initialized.
2014-08-26T15:14:47.090Z - info: Kalabox Version: 1.0-beta7
2014-08-26T15:14:47.090Z - info: Mac OS Version: 10.9.4

2014-08-26T15:14:47.090Z - info: VirtualBox Version: 4.3.6r91406

2014-08-26T15:14:47.091Z - info: Vagrant Version: Vagrant 1.4.3

2014-08-26T15:14:49.922Z - warn: Error retrieving sites list: Error: connect ECONNREFUSED
2014-08-26T15:19:17.572Z - warn: Error retrieving sites list: Error: connect ECONNREFUSED
2014-08-26T15:19:23.340Z - error: An error occurred in an async call.
cause stack is ...
  Error: Command failed: PHP Fatal error:  Call to undefined function _terminatur_keys() in /usr/share/drush/commands/core/core.drush.inc(637) : eval()'d code on line 1
Drush command terminated abnormally due to an unrecoverable error.       [error]
Error: Call to undefined function _terminatur_keys() in
/usr/share/drush/commands/core/core.drush.inc(637) : eval()'d code,
line 1

```
at ChildProcess.exithandler (child_process.js:661:12)
at ChildProcess.EventEmitter.emit (events.js:101:17)
at maybeClose (child_process.js:773:16)
at Socket.eval (child_process.js:994:11)
at Socket.EventEmitter.emit (events.js:98:17)
at Pipe.eval [as close] (net.js:459:12)
```

async call history is ...
 [ { function: 'startBox4',
    location: '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.ml40sl/kalabox/box.js:168:108',
    mapping: { description: 'tail' } } ]

Maybe vm customizing has not been done during Kalabox installation?

**Comment by pirog** [comment]
Yeah that's my guess too. In your browser can you go to start.kala or
php.kala?

If you can't I would try reinstalling.

On Tuesday, August 26, 2014, César García notifications@github.com wrote:

> Hi @pirog https://github.com/pirog,
> 
> That was exactly my same thought. I've just logged in into the vm using
> ssh. This is the output:
> 
> The authenticity of host '[127.0.0.1]:2222 ([127.0.0.1]:2222)' can't be
> established.
> RSA key fingerprint is 50:db:75:ba:11:2f:43:c9[image: :ab:]
> 14:40:6d:7f:a1:ee:e3.
> Are you sure you want to continue connecting (yes/no)? yes
> Warning: Permanently added '[127.0.0.1]:2222' (RSA) to the list of known
> hosts.
> Welcome to Ubuntu 12.04 LTS (GNU/Linux 3.2.0-23-generic x86_64)
> - Documentation: https://help.ubuntu.com/ New release '14.04.1 LTS'
>   available. Run 'do-release-upgrade' to upgrade to it.
> 
> Welcome to your Vagrant-built virtual machine.
> Last login: Fri Sep 14 06:23:18 2012 from 10.0.2.2
> vagrant@kala:~$ drush
> The program 'drush' is currently not installed. You can install it by
> typing:
> sudo apt-get install drush
> vagrant@kala:~$ sudo apt-get install drush
> Reading package lists... Done
> Building dependency tree
> 
> Reading state information... Done
> The following extra packages will be installed:
> autoconf automake autotools-dev libltdl-dev libltdl7 libmysqlclient18
> libtool m4 mysql-common php-console-table php-pear php5-cli php5-common
> php5-dev php5-mysql shtool
> Suggested packages:
> autoconf2.13 autoconf-archive gnu-standards autoconf-doc gettext
> mysql-client libtool-doc automaken gfortran fortran95-compiler gcj
> php5-suhosin
> The following NEW packages will be installed:
> autoconf automake autotools-dev drush libltdl-dev libltdl7 libmysqlclient18
> libtool m4 mysql-common php-console-table php-pear php5-cli php5-common
> php5-dev php5-mysql shtool
> 0 upgraded, 17 newly installed, 0 to remove and 169 not upgraded.
> Need to get 8,019 kB/8,981 kB of archives.
> After this operation, 28.0 MB of additional disk space will be used.
> Do you want to continue [Y/n]?
> Get:1 http://us.archive.ubuntu.com/ubuntu/ precise/main libltdl7 amd64
> 2.4.2-1ubuntu1 [37.6 kB]
> Get:2 http://us.archive.ubuntu.com/ubuntu/ precise/main m4 amd64
> 1.4.16-2ubuntu1 [200 kB]
> Get:3 http://us.archive.ubuntu.com/ubuntu/ precise/main autoconf all
> 2.68-1ubuntu2 [560 kB]
> Get:4 http://us.archive.ubuntu.com/ubuntu/ precise/main autotools-dev all
> 20120210.1ubuntu1 [42.4 kB]
> Get:5 http://us.archive.ubuntu.com/ubuntu/ precise/main automake all
> 1:1.11.3-1ubuntu2 [571 kB]
> Get:6 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main
> php5-common amd64 5.3.10-1ubuntu3.13 [1,774 kB]
> Get:7 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php5-cli
> amd64 5.3.10-1ubuntu3.13 [3,051 kB]
> Get:8 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php-pear
> all 5.3.10-1ubuntu3.13 [371 kB]
> Get:9 http://us.archive.ubuntu.com/ubuntu/ precise/universe
> php-console-table all 1.1.4-1 [14.5 kB]
> Get:10 http://us.archive.ubuntu.com/ubuntu/ precise/universe drush all
> 4.5-6 [242 kB]
> Get:11 http://us.archive.ubuntu.com/ubuntu/ precise/main libltdl-dev
> amd64 2.4.2-1ubuntu1 [202 kB]
> Get:12 http://us.archive.ubuntu.com/ubuntu/ precise/main libtool amd64
> 2.4.2-1ubuntu1 [302 kB]
> Get:13 http://us.archive.ubuntu.com/ubuntu/ precise/main shtool all
> 2.0.8-6 [149 kB]
> Get:14 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main php5-dev
> amd64 5.3.10-1ubuntu3.13 [424 kB]
> Get:15 http://us.archive.ubuntu.com/ubuntu/ precise-updates/main
> php5-mysql amd64 5.3.10-1ubuntu3.13 [76.6 kB]
> Fetched 8,019 kB in 8s (983 kB/s)
> 
> Selecting previously unselected package libltdl7.
> (Reading database ... 51754 files and directories currently installed.)
> Unpacking libltdl7 (from .../libltdl7_2.4.2-1ubuntu1_amd64.deb) ...
> Selecting previously unselected package mysql-common.
> Unpacking mysql-common (from
> .../mysql-common_5.5.38-0ubuntu0.12.04.1_all.deb) ...
> Selecting previously unselected package libmysqlclient18.
> Unpacking libmysqlclient18 (from
> .../libmysqlclient18_5.5.38-0ubuntu0.12.04.1_amd64.deb) ...
> Selecting previously unselected package m4.
> Unpacking m4 (from .../m4_1.4.16-2ubuntu1_amd64.deb) ...
> Selecting previously unselected package autoconf.
> Unpacking autoconf (from .../autoconf_2.68-1ubuntu2_all.deb) ...
> Selecting previously unselected package autotools-dev.
> Unpacking autotools-dev (from .../autotools-dev_20120210.1ubuntu1_all.deb)
> ...
> Selecting previously unselected package automake.
> Unpacking automake (from .../automake_1%3a1.11.3-1ubuntu2_all.deb) ...
> Selecting previously unselected package php5-common.
> Unpacking php5-common (from .../php5-common_5.3.10-1ubuntu3.13_amd64.deb)
> ...
> Selecting previously unselected package php5-cli.
> Unpacking php5-cli (from .../php5-cli_5.3.10-1ubuntu3.13_amd64.deb) ...
> Selecting previously unselected package php-pear.
> Unpacking php-pear (from .../php-pear_5.3.10-1ubuntu3.13_all.deb) ...
> Selecting previously unselected package php-console-table.
> Unpacking php-console-table (from .../php-console-table_1.1.4-1_all.deb)
> ...
> Selecting previously unselected package drush.
> Unpacking drush (from .../archives/drush_4.5-6_all.deb) ...
> Selecting previously unselected package libltdl-dev.
> Unpacking libltdl-dev (from .../libltdl-dev_2.4.2-1ubuntu1_amd64.deb) ...
> Selecting previously unselected package libtool.
> Unpacking libtool (from .../libtool_2.4.2-1ubuntu1_amd64.deb) ...
> Selecting previously unselected package shtool.
> Unpacking shtool (from .../shtool_2.0.8-6_all.deb) ...
> Selecting previously unselected package php5-dev.
> Unpacking php5-dev (from .../php5-dev_5.3.10-1ubuntu3.13_amd64.deb) ...
> Selecting previously unselected package php5-mysql.
> Unpacking php5-mysql (from .../php5-mysql_5.3.10-1ubuntu3.13_amd64.deb) ...
> Processing triggers for install-info ...
> Processing triggers for man-db ...
> Setting up libltdl7 (2.4.2-1ubuntu1) ...
> Setting up mysql-common (5.5.38-0ubuntu0.12.04.1) ...
> Setting up libmysqlclient18 (5.5.38-0ubuntu0.12.04.1) ...
> Setting up m4 (1.4.16-2ubuntu1) ...
> Setting up autoconf (2.68-1ubuntu2) ...
> Setting up autotools-dev (20120210.1ubuntu1) ...
> Setting up automake (1:1.11.3-1ubuntu2) ...
> update-alternatives: using /usr/bin/automake-1.11 to provide
> /usr/bin/automake (automake) in auto mode.
> Setting up php5-common (5.3.10-1ubuntu3.13) ...
> Setting up php5-cli (5.3.10-1ubuntu3.13) ...
> 
> Creating config file /etc/php5/cli/php.ini with new version
> update-alternatives: using /usr/bin/php5 to provide /usr/bin/php (php) in
> auto mode.
> Setting up php-pear (5.3.10-1ubuntu3.13) ...
> Setting up php-console-table (1.1.4-1) ...
> Setting up drush (4.5-6) ...
> Setting up libltdl-dev (2.4.2-1ubuntu1) ...
> Setting up libtool (2.4.2-1ubuntu1) ...
> Setting up shtool (2.0.8-6) ...
> Setting up php5-dev (5.3.10-1ubuntu3.13) ...
> update-alternatives: using /usr/bin/php-config5 to provide
> /usr/bin/php-config (php-config) in auto mode.
> update-alternatives: using /usr/bin/phpize5 to provide /usr/bin/phpize
> (phpize) in auto mode.
> Setting up php5-mysql (5.3.10-1ubuntu3.13) ...
> Processing triggers for libc-bin ...
> ldconfig deferred processing now taking place
> vagrant@kala:~$ drush
> Execute a drush command. Run drush help [command] to view command-specific
> help. Run drush topic to read even more documentation.
> 
> Global options (see drush topic core-global-options for the full list).:
> -r , --root= Drupal root directory to use
> 
> (default: current directory)
> 
> -l http://example.com:8888, URI of the drupal site to use (only
> 
> --uri=http://example.com:8888 needed in multisite environments or
> 
> when running on an alternate port)
> 
> -v, --verbose Display extra information about the
> 
> command.
> 
> -d, --debug Display even more information,
> 
> including internal messages.
> 
> -y, --yes Assume 'yes' as answer to all
> 
> prompts
> 
> -n, --no Assume 'no' as answer to all prompts
> -s, --simulate Simulate all relevant actions (don't
> actually change the system)
> 
> -p, --pipe Emit a compact representation of the
> command for scripting.
> 
> -h, --help This help system.
> 
> --version Show drush version.
> 
> --php The absolute path to your PHP
> 
> intepreter, if not 'php' in the
> 
> path.
> 
> Core drush commands: (core)
> archive-dump (ard, Backup your code, files, and database into a single
> 
> archive-backup, arb) file.
> 
> archive-restore Expand a site archive into a Drupal web site.
> 
> (arr)
> 
> cache-clear (cc) Clear a specific cache, or all drupal caches.
> 
> cache-get (cg) Fetch a cached object and display it.
> 
> cache-set (cs) Cache an object expressed in JSON or var_export()
> 
> format.
> 
> core-cli (cli) Enter a new shell optimized for drush use.
> 
> core-cron (cron) Run all cron hooks in all active modules for specified
> 
> site.
> 
> core-rsync (rsync) Rsync the Drupal tree to/from another server using ssh.
> 
> core-status (status, Provides a birds-eye view of the current Drupal
> 
> st) installation, if any.
> 
> core-topic (topic) Read detailed documentation on a given topic.
> 
> drupal-directory Return path to a given module/theme directory.
> 
> (dd)
> 
> help Print this help message. See drush help help for more
> 
> options.
> 
> image-flush Flush all derived images for a given style.
> 
> php-eval (eval, ev) Evaluate arbitrary php code after bootstrapping Drupal
> 
> (if available).
> 
> php-script (scr) Run php script(s).
> 
> search-index Index the remaining search items without wiping the
> 
> index.
> 
> search-reindex Force the search index to be rebuilt.
> 
> search-status Show how many items remain to be indexed out of the
> 
> total.
> 
> self-update Update drush to the latest version, if available.
> 
> (selfupdate)
> 
> site-alias (sa) Print site alias records for all known site aliases and
> 
> local sites.
> 
> site-install (si) Install Drupal along with modules/themes/configuration
> 
> using the specified install profile.
> 
> site-upgrade (sup) Run a major version upgrade for Drupal (e.g. Drupal 6
> to
> Drupal 7). A copy of the site is made, and then
> 
> upgraded; the source site is not changed.
> 
> test-clean Clean temporary tables and files.
> 
> test-run Run tests. Note that you must use the --uri option.
> 
> updatedb (updb) Apply any database updates required (as with running
> 
> update.php).
> 
> variable-delete Delete a variable.
> 
> (vdel)
> 
> variable-get (vget) Get a list of some or all site variables and values.
> 
> variable-set (vset) Set a variable.
> 
> version Show drush version.
> 
> watchdog-delete Delete watchdog messages.
> 
> (wd-del, wd-delete)
> 
> watchdog-list Show available message types and severity levels. A
> 
> (wd-list) prompt will ask for a choice to show watchdog messages.
> 
> watchdog-show Show watchdog messages.
> 
> (wd-show, ws)
> 
> Field commands: (field)
> field-clone Clone a field and all its instances.
> 
> field-create Create fields and instances. Returns urls for field
> 
> editing.
> 
> field-delete Delete a field and its instances.
> 
> field-info View information about fields, field_types, and widgets.
> field-update Return URL for field editing web page.
> 
> Project manager commands: (pm)
> pm-disable (dis) Disable one or more extensions (modules or themes).
> 
> pm-download (dl) Download projects from drupal.org or other sources.
> 
> pm-enable (en) Enable one or more extensions (modules or themes).
> 
> pm-info (pmi) Show detailed info for one or more extensions (modules
> or themes).
> 
> pm-list (pml) Show a list of available extensions (modules and
> 
> themes).
> 
> pm-refresh (rf) Refresh update status information.
> 
> pm-releasenotes Print release notes for given projects.
> 
> (rln)
> 
> pm-releases (rl) Print release information for given projects.
> 
> pm-uninstall Uninstall one or more modules.
> 
> pm-update (up) Update Drupal core and contrib projects and apply any
> 
> pending database updates (Same as pm-updatecode +
> 
> updatedb).
> 
> pm-updatecode (upc) Update Drupal core and contrib projects to latest
> 
> recommended releases.
> 
> SQL commands: (sql)
> sql-cli (sqlc) Open a SQL command-line interface using Drupal's
> 
> credentials.
> 
> sql-connect A string for connecting to the DB.
> 
> sql-drop Drop all tables in a given database.
> 
> sql-dump Exports the Drupal DB as SQL using mysqldump or
> 
> equivalent.
> 
> sql-query (sqlq) Execute a query against the site database.
> 
> sql-sync Copy and import source database to target database.
> Transfers via rsync.
> 
> User commands: (user)
> user-add-role (urol) Add a role to the specified user accounts.
> 
> user-block (ublk) Block the specified user(s).
> 
> user-cancel (ucan) Cancel a user account with the specified name.
> 
> user-create (ucrt) Create a user account with the specified name.
> 
> user-information Print information about the specified user(s).
> 
> (uinf)
> 
> user-login (uli) Display a one time login link for the given user account
> (defaults to uid 1).
> 
> user-password (upwd) (Re)Set the password for the user account with the
> 
> specified name.
> 
> user-remove-role Remove a role from the specified user accounts.
> 
> (urrol)
> 
> user-unblock (uublk) Unblock the specified user(s).
> 
> vagrant@kala:~$
> 
> This is the error log once I stop and start kalabox again.
> Kalabox Log
> 
> 2014-08-24T22:47:04.233Z - info: Box initialized.
> 2014-08-24T22:47:04.237Z - info: Kalabox Version: 1.0-beta7
> 2014-08-24T22:47:04.237Z - info: Mac OS Version: 10.9.4
> 
> 2014-08-24T22:47:04.238Z - info: VirtualBox Version: false
> 2014-08-24T22:47:04.238Z - info: Vagrant Version: false
> 2014-08-24T22:48:03.034Z - info: VirtualBox not installed or wrong
> version. Installing.
> 2014-08-24T22:48:03.037Z - info: User granted permission to install
> VirtualBox
> 2014-08-24T22:49:03.409Z - info: Successfully downloaded VirtualBox DMG.
> 2014-08-24T22:49:29.569Z - info: Successfully mounted VirtualBox DMG.
> 2014-08-24T22:50:06.992Z - info: VirtualBox installation complete.
> 2014-08-24T22:50:08.475Z - info: Successfully ejected VirtualBox DMG.
> 2014-08-24T22:50:08.489Z - info: Vagrant not installed or wrong version.
> Installing.
> 2014-08-24T22:50:08.491Z - info: User granted permission to install Vagrant
> 2014-08-24T22:50:24.254Z - info: Successfully downloaded Vagrant DMG.
> 2014-08-24T22:50:32.266Z - info: Successfully mounted Vagrant DMG.
> 2014-08-24T22:50:45.532Z - info: Vagrant installation complete.
> 2014-08-24T22:50:45.868Z - info: Successfully ejected Vagrant DMG.
> 2014-08-24T22:52:09.180Z - info: Box image downloaded.
> 2014-08-24T22:52:09.843Z - info: Installing Vagrant plugin
> vagrant-hostsupdater
> 2014-08-24T22:52:12.826Z - info: Kalastack downloaded
> 2014-08-24T22:52:12.828Z - info: Installing Vagrant plugin
> vagrant-hostsupdater
> 2014-08-24T22:52:36.505Z - info: Adding base box image to Vagrant.
> 2014-08-24T22:52:50.409Z - info: Wrote configuration file.
> 2014-08-24T22:52:50.610Z - info: Executing box spinup.
> 2014-08-24T22:52:50.725Z - info: Starting spinup attempt 0
> 2014-08-24T23:42:21.252Z - info: Box initialized.
> 2014-08-24T23:42:21.259Z - info: Kalabox Version: 1.0-beta7
> 2014-08-24T23:42:21.311Z - info: Mac OS Version: 10.9.4
> 
> 2014-08-24T23:42:21.311Z - info: VirtualBox Version: 4.3.6r91406
> 
> 2014-08-24T23:42:21.351Z - info: Vagrant Version: Vagrant 1.4.3
> 
> 2014-08-24T23:42:21.360Z - info: Installation completed successfully.
> 2014-08-24T23:42:34.968Z - info: Existing VirtualBox installation, version
> 4.3.6
> 2014-08-24T23:42:34.971Z - info: Existing Vagrant installation, version
> 1.4.3
> 2014-08-24T23:42:34.991Z - info: Kalabox image already downloaded.
> 2014-08-24T23:42:34.992Z - info: Box image downloaded.
> 2014-08-24T23:42:35.126Z - info: Kalastack already downloaded
> 2014-08-24T23:42:35.127Z - info: Installing Vagrant plugin
> vagrant-hostsupdater
> 2014-08-24T23:42:37.814Z - warn: Error retrieving sites list: Error:
> getaddrinfo ENOTFOUND
> 2014-08-24T23:42:53.943Z - info: Kalastack downloaded
> 2014-08-24T23:42:53.945Z - info: Installing Vagrant plugin
> vagrant-hostsupdater
> 2014-08-24T23:43:20.086Z - info: Wrote configuration file.
> 2014-08-24T23:43:20.204Z - info: Executing box spinup.
> 2014-08-24T23:43:20.303Z - info: Starting spinup attempt 0
> 2014-08-24T23:43:38.992Z - info: Box initialized.
> 2014-08-24T23:43:38.993Z - info: Kalabox Version: 1.0-beta7
> 2014-08-24T23:43:38.993Z - info: Mac OS Version: 10.9.4
> 
> 2014-08-24T23:43:38.993Z - info: VirtualBox Version: 4.3.6r91406
> 
> 2014-08-24T23:43:38.993Z - info: Vagrant Version: Vagrant 1.4.3
> 
> 2014-08-24T23:43:38.995Z - info: Installation completed successfully.
> 2014-08-24T23:45:41.569Z - warn: Error retrieving sites list: Error:
> getaddrinfo ENOTFOUND
> 2014-08-24T23:47:13.879Z - warn: Error retrieving sites list: Error:
> connect ECONNREFUSED
> 2014-08-24T23:47:19.353Z - error: An error occurred in an async call.
> cause stack is ...
> Error: Command failed: bash: drush: command not found
> 
> at ChildProcess.exithandler (child_process.js:661:12)
> at ChildProcess.EventEmitter.emit (events.js:101:17)
> at maybeClose (child_process.js:773:16)
> at Socket.eval (child_process.js:994:11)
> at Socket.EventEmitter.emit (events.js:98:17)
> at Pipe.eval [as close](net.js:459:12)
> 
> async call history is ...
> [ { function: 'startBox4',
> location:
> '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.lY24yP/kalabox/box.js:168:108',
> mapping: { description: 'tail' } } ]
> 2014-08-25T22:44:04.056Z - info: Box initialized.
> 2014-08-25T22:44:04.061Z - info: Kalabox Version: 1.0-beta7
> 2014-08-25T22:44:04.062Z - info: Mac OS Version: 10.9.4
> 
> 2014-08-25T22:44:04.062Z - info: VirtualBox Version: 4.3.6r91406
> 
> 2014-08-25T22:44:04.062Z - info: Vagrant Version: Vagrant 1.4.3
> 
> 2014-08-25T22:44:07.558Z - warn: Error retrieving sites list: Error:
> connect ECONNREFUSED
> 2014-08-25T22:47:20.693Z - warn: Box has no Internet connection.
> 2014-08-25T22:53:27.459Z - warn: Error retrieving sites list: Error:
> connect ECONNREFUSED
> 2014-08-25T22:53:31.621Z - error: An error occurred in an async call.
> cause stack is ...
> Error: Command failed: bash: drush: command not found
> 
> at ChildProcess.exithandler (child_process.js:661:12)
> at ChildProcess.EventEmitter.emit (events.js:101:17)
> at maybeClose (child_process.js:773:16)
> at Socket.eval (child_process.js:994:11)
> at Socket.EventEmitter.emit (events.js:98:17)
> at Pipe.eval [as close](net.js:459:12)
> 
> async call history is ...
> [ { function: 'startBox4',
> location:
> '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.sK7HIv/kalabox/box.js:168:108',
> mapping: { description: 'tail' } } ]
> 2014-08-26T15:14:47.087Z - info: Box initialized.
> 2014-08-26T15:14:47.090Z - info: Kalabox Version: 1.0-beta7
> 2014-08-26T15:14:47.090Z - info: Mac OS Version: 10.9.4
> 
> 2014-08-26T15:14:47.090Z - info: VirtualBox Version: 4.3.6r91406
> 
> 2014-08-26T15:14:47.091Z - info: Vagrant Version: Vagrant 1.4.3
> 
> 2014-08-26T15:14:49.922Z - warn: Error retrieving sites list: Error:
> connect ECONNREFUSED
> 2014-08-26T15:19:17.572Z - warn: Error retrieving sites list: Error:
> connect ECONNREFUSED
> 2014-08-26T15:19:23.340Z - error: An error occurred in an async call.
> cause stack is ...
> Error: Command failed: PHP Fatal error: Call to undefined function
> _terminatur_keys() in /usr/share/drush/commands/core/core.drush.inc(637) :
> eval()'d code on line 1
> Drush command terminated abnormally due to an unrecoverable error. [error]
> Error: Call to undefined function _terminatur_keys() in
> /usr/share/drush/commands/core/core.drush.inc(637) : eval()'d code,
> line 1
> 
> at ChildProcess.exithandler (child_process.js:661:12)
> at ChildProcess.EventEmitter.emit (events.js:101:17)
> at maybeClose (child_process.js:773:16)
> at Socket.eval (child_process.js:994:11)
> at Socket.EventEmitter.emit (events.js:98:17)
> at Pipe.eval [as close](net.js:459:12)
> 
> async call history is ...
> [ { function: 'startBox4',
> location:
> '/private/var/folders/s7/xmb361351cnf_n9dj_98ncb80000gn/T/.org.chromium.Chromium.ml40sl/kalabox/box.js:168:108',
> mapping: { description: 'tail' } } ]
> 
> Maybe vm customizing has not been done during Kalabox installation?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kalamuna/kalaboxv1/issues/122#issuecomment-53438353.

## 

Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com
